src/main/java/com/example/carservice/controller/CarController.java

```diff
@@ -13,13 +12,20 @@
 public class CarController {
 
     private final CarService service;
+    private final CarRepository carRepository;
```

Comment:
Repository need to move to Service.

Flow: Controller -> Service -> Repository

Reply:
Repository moved to service.